src/set-pages-config.js

```diff
@@ -85,7 +101,7 @@ function setPagesConfig({ staticSiteGenerator, generatorConfigFile, siteUrl }) {
       core.warning(
         `Unsupported configuration file extension. Currently supported extensions: ${SUPPORTED_FILE_EXTENSIONS.map(
           ext => JSON.stringify(ext)
-        ).join(', ')}`,
+        ).join(', ')} recieved: something`,
```

Comment:
Looks like a copy/paste misap? 

```suggestion
        ).join(', ')}`,
```